src/data/community_de.json

```diff
@@ -4,7 +4,9 @@
             "title": "Los geht's",
             "textblock": [
                 "Willkommen in der Corona-Warn-App-Community!",
-                "Wenn Du mehr darüber erfahren möchtest, wie Du bei uns mitmachen kannst, bist Du hier genau richtig. Nachfolgend findest Du Infos, wie Du bei uns mitarbeiten und zum Code und der Dokumentation der Corona-Warn-App beitragen kannst. Los geht's!"
+                "Wenn Du mehr darüber erfahren möchtest, wie Du bei uns mitmachen kannst, bist Du hier genau richtig. Nachfolgend findest Du Infos, wie Du bei uns mitarbeiten und zum Code und der Dokumentation der Corona-Warn-App beitragen kannst.",
+                "Du kannst uns auch auf Twitter und Instagram <a href='/de/community/#social-media'>folgen</a>, dich über die neusten Entwicklungen informieren und dich mit den anderen Followern unterhalten.",
```

Comment:
The German text should be consistent about whether du / dich is written capitalized or not.

See the advice from Duden https://www.duden.de/sprachwissen/sprachratgeber/Gross-oder-Kleinschreibung-von-duDu-und-ihrIhr. 

I suggest following from the existing text and writing the pronouns capitalized.

Reply:
Thanks, we follow your suggestion and write the pronouns capitalized.